src/Background/stat_estimators.jl

```diff
@@ -1,5 +1,44 @@
 using Statistics
 
+"""
+    sigma_clip(data, sigma_low, sigma_high; center=median, std=std, iterations=5)
+    sigma_clip(data, sigma; center=median, std=std, iteration=5)
```

Comment:
I see no point in having this iterations keyword and functionality. If someone wants to repeat a function, there are simple ways to do that outside of the function.